src/assets/styles/_colors_redesign.less

```diff
@@ -13,16 +13,18 @@
 @color-secondary-background: #120D1B;
 @color-tertiary-background: #1B1528;
 @color-theme-primary: #211A32;
+@color-yellowish-alert: #FBE9A6;
```

Comment:
don't think you are supposed to have color names in here?

Reply:
Hows that. Changed to info-message